Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -1275,11 +1278,15 @@ - (void)tableView:(__attribute__((unused)) UITableView *)tableView
       // Collapse barcode by adjusting certain constraints
       if (self.barcodeImageView.bounds.size.height > 0) {
         self.barcodeHeightConstraint.constant = 0.0;
+        self.barcodeTextHeightConstraint.constant = 0.0;
+        self.barcodeTextLabelSpaceConstraint.constant = 0.0;
         self.barcodeLabelSpaceConstraint.constant = 0.0;
         self.barcodeImageLabel.text = NSLocalizedString(@"Show Barcode", nil);
         [[UIScreen mainScreen] setBrightness:self.userBrightnessSetting];
       } else {
         self.barcodeHeightConstraint.constant = 100.0;
+        self.barcodeTextHeightConstraint.constant = 30.0;
+        self.barcodeTextLabelSpaceConstraint.constant = -12.0;
```

Comment:
just wondering if we can put some of these numbers in constants instead of repeating them... like this one seems related to a spacer distance.